resources/views/themes/default1/front/checkout.blade.php

```diff
@@ -316,30 +316,7 @@
 </div>
 </div>
 @elseif (\Cart::isEmpty())
-    <div class="featured-boxes">
```

Comment:
Why it has been removed?

Reply:
I've removed this because I've made another box with some little bit different ui. But i don't change the featured-boxes css as I don't want to change it in the whole system.